docs/core/install/linux-ubuntu.md

```diff
@@ -1,200 +1,181 @@
 ---
-title: Install .NET on Ubuntu
+title: .NET and Ubuntu overview
 description: Demonstrates the various ways to install .NET SDK and .NET Runtime on Ubuntu.
 author: adegeo
 ms.author: adegeo
-ms.date: 12/21/2022
+ms.date: 02/17/2023
 ---
 
 # Install the .NET SDK or the .NET Runtime on Ubuntu
 
-.NET is supported on Ubuntu. This article describes how to install .NET on Ubuntu. When an [Ubuntu version](https://wiki.ubuntu.com/Releases) falls out of support, .NET is no longer supported with that version.
+This article describes how to install .NET on Ubuntu. For each version of Ubuntu, the Microsoft package repository contains every version of .NET that is currently, or was previously, supported. Some versions of Ubuntu contain .NET in the Ubuntu package feed. For more information about available versions, see the [Supported distributions](#supported-distributions) section.
 
-[!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
+> [!WARNING]
+> It's recommended that you avoid using both feeds for .NET, as this leads to problems when apps try to resolve a specific version of .NET. Mixing the package feeds for .NET is an advanced scenario, and is covered in [Troubleshoot .NET errors related to missing files on Linux - Solutions](linux-package-mixup.md).
 
-[!INCLUDE [linux-install-package-manager-x64-vs-arm](includes/linux-install-package-manager-x64-vs-arm.md)]
+| Method | Pros | Cons |
+|--------|------|------|
+| [Package manager<br>(Microsoft feed)*](#register-the-microsoft-package-repository) | <ul><li>Supported versions always available.</li><li>Preview releases are available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>Requires registering the Microsoft package repository.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Package manager<br>(Ubuntu feed)](#supported-distributions) | <ul><li>Usually the latest version is available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>.NET versions available vary by Ubuntu version.</li><li>Preview releases aren't available.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Snap](linux-snap.md) | <ul><li>Dependencies are included.</li><li>All supported CPU architectures available.</li><li>Easy to update to the latest patch version.</li><li>Easy removal.</li></ul> | <ul><li>Cumbersome to install and enable multiple runtimes.</li></ul> |
+| [Script \ Manual extraction](linux-scripted-manual.md) | <ul><li>Control where .NET is installed.</li></ul> | <ul><li>Manually install updates.</li><li>Manually install dependencies.</li><li>Manual removal.</li></ul> |
 
-## Supported distributions
+\* This is the recommended installation method for end-users and developers.
 
-The following table is a list of currently supported .NET releases and the versions of Ubuntu they're supported on.
+## Decide how to install .NET
```

Comment:
I think there are more factors that one may need to consider when deciding between Microsoft vs. Ubuntu feeds:

- Primary support provider: Primary support provider for Microsoft feeds is Microsoft. Primary support provider for Ubuntu feeds is Canonical. The enterprise may prefer one or the other based on the support contracts in place.
- Performance characteristics: Performance characteristics of the binaries from Microsoft feeds and binaries from Ubuntu feeds may be different (depends on the workload).

Reply:
Yes, the Ubuntu feed binaries are built by Canonical.

Reply:
@jkotas Is there a link that describes this? I want to understand how I should phrase this.

Reply:
https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages touches on this: .NET is [supported by Microsoft](https://github.com/dotnet/core/blob/main/microsoft-support.md) when downloaded from a Microsoft source. Best effort support is offered from Microsoft when downloaded from elsewhere.

Reply:
@jkotas I'm sorry, I meant something that details how Canonical's packages would differ from Microsoft's. I can't really say that perf may differ between the two builds without knowing how or referencing something. otherwise, it's purely speculation. Someone may want to know if it's a negative or positive perf change.

Reply:
We do not have comprehensive data about this. I have noticed some of these differences when looking around. For example, `libcoreclr.so` size in Microsoft build is 7,355,856. `libcoreclr.so` size in Canonical build is 6,967,488. It is more than 5% difference in binary size. 

You can skip mentioning perf as one of the differences. 

It would be useful to have comparison of the two options. The decision matrix tells people what to do and most of the options tell you to try installing from Ubuntu feeds first, and switch to Microsoft feed if that did not work.

I think it would be useful to explain the benefits of each feed. The decision matrix tells people to use one or the other, but it does not tell them why. What am I going to lose if I am always going to use the Microsoft feeds? Yes, there is an extra step, but I do not have to make any choices and troubleshooting.

Reply:
Some users might also care about telemetry as a difference. Microsoft's builds include/enable telemetry in the .NET SDK (opt-out is a manual process). Builds from other vendors (including Canonical) generally disable it.